calculator.cpp

```diff
@@ -2,26 +2,59 @@
 using namespace std;
 
 int addition(int number_1, int number_2){
+return number_1+number_2;
 
 }
 
 int subtraction(int number_1, int number_2){
-
+return number_1-number_2;
 }
 
 int multiplication(int number_1, int number_2){
-
+return number_1*number_2;
 }
 
 int division(int number_1, int number_2){
-
+if(num2==0)
+cout<<"DIVIDE BY ZERO ERROR";
+else
+return number_1/number_2;
 }
 
 int get_input(){
+int operand1,operand2;
+char operator;
```

Comment:
operator is c++ key word
[https://link.springer.com/content/pdf/bbm%3A978-1-4471-0289-2%2F1.pdf](url)